contracts/docs/interchainqueries/howto/register_custom_kv_icq/README.md

```diff
@@ -0,0 +1,3 @@
+# HowTo register a KV Interchain Query with custom keys contract
```

Comment:
What is the main goal of the custom contracts. What does the contract teach you what non custom does not. Answer for those questions may be useful in the README

Reply:
these questions are answered in the documentation by the link